src/Testcontainers.WebDriver/WebDriverBuilder.cs

```diff
@@ -0,0 +1,95 @@
+namespace WebDriver;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class WebDriverBuilder : ContainerBuilder<WebDriverBuilder, WebDriverContainer, WebDriverConfiguration>
+{
+    public const string ChromeStandaloneImage = "selenium/standalone-chrome";
+    public const string FirefoxStandaloneImage = "selenium/standalone-firefox";
+    public const string EdgeStandaloneImage = "selenium/standalone-edge";
+    public const string OperaStandaloneImage = "selenium/standalone-opera";
```

Comment:
In Testcontainers for Java, the approach is to declare which images are compatible with the Testcontainers class implementation and if there is a custom image then `DockerImageName.parse("my-custom-image").asCompatibleSubstituteFor("mysql")`. I think there is not similar support in dotnet, yet. IMO, dotnet should follow its own approach until the same support to Java is added to the project.

Reply:
The way we do it in Java is a bit uncommon for this class, since we infer the image from the webdriver capabilities:
https://github.com/testcontainers/testcontainers-java/blob/77a3c3726500dc07159762cfddc052596e27f420/modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java#L260-L274

We don't need to replicate this logic in the .NET implementation.

Reply:
That is what I think to do - the [answer ](https://github.com/testcontainers/testcontainers-dotnet/pull/780#discussion_r1106900073) for @HofmeisterAn.
I think also to not replicate this logic in the .NET implementation.
So maybe we **don't** assert the image name and version from compatibility - let the developer - the client does it ?